tests/main/install-hook-can-run-svcs/test-snapd-install-hook-runs-svc/bin/svc.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -e
+
+echo "snap-svc-did-this" > "$SNAP_DATA/data"
+
+sleep 1000
```

Comment:
Please use `exec sleep` to avoid the shell process to stick around.

Reply:
but won't the shell process be killed when we `snapctl` stop it anyways?

Reply:
well I did it anyways in case there is some reason that this is preferable